scrapy/settings/default_settings.py

```diff
@@ -245,6 +245,7 @@
 SCHEDULER_DISK_QUEUE = 'scrapy.squeues.PickleLifoDiskQueue'
 SCHEDULER_MEMORY_QUEUE = 'scrapy.squeues.LifoMemoryQueue'
 SCHEDULER_PRIORITY_QUEUE = 'queuelib.PriorityQueue'
+SCHEDULER_PREFER_MEMORY_QUEUE = True
```

Comment:
What about encouraging to set this to `False` in new spiders through `genspider`, similar to how it was [done for `ROBOTSTXT_OBEY`](https://github.com/scrapy/scrapy/blob/master/scrapy/templates/project/module/settings.py.tmpl#L22)?

Reply:
I don't really have a strong opinion on it, I'm happy with either value as default as long as the behaviour is consistent and documented.

Reply:
I think that'd be good to discuss it in a separate issue / pull request, because the best way is unclear, and the way @elacuesta uses is "safe" and backwards compatible.